### DISCUSSION TITLE
Is it possible to hide stories in Storybook on the sidebar? But still be able to access them by URL?

### DISCUSSION THREAD

**Discussion by TonyHoanTrinh** [storybookjs/storybook#15425] [Help]
I'm trying to do some stuff in Storybook. I have a component called Button and there are too many stories on the sidebar.

![image](https://user-images.githubusercontent.com/26545361/123850131-a3146580-d8e7-11eb-809e-c640ae3d0e61.png)

Is there a way to be able to "hide" the stories on the sidebar but having the stories still loading in? So basically for example if I could hide the "With Text Only" story but have the story is loaded in so you could still access it at the url "http://localhost:9001/?path=/story/ccfk-%F0%9F%A7%A9-%E2%A0%80components-button--with-text-only" or access it using addon-links.

I'm unsure if this is possible or not and would really appreciate any help, thank you!

**Comment by shilman** [comment]
Nope, but there's an open issue here https://github.com/storybookjs/storybook/issues/9209
